File: Multiples_of_3_or_5/Python/main.py
def list_mult(n:int):
    mult = [x for x in range(1,n) if x % 3 == 0 or x % 5 == 0]
    return sum(mult)

if __name__ == '__main__':
    n = int(input('Type the max number:'))
    res = list_mult(n)
    print(f'The sum of the numbers below \033[32m{n}\033[0m that are multiples of 3 or 5 is \033[32m{res}\033[0m ')

File: Multiples_of_3_or_5/Golang/main.go
package main
import "fmt"

func main() {
	var max int
	var sum int
	fmt.Println("Type the max value:")
	fmt.Scan(&max)
	for i := 1; i<max; i++{
		if verifyMult(i){
			sum += i
		}
	}
	fmt.Printf("The sum of the multiples of 3 or 5 below \033[32m %d \033[0m is \033[32m %d \033[0m",max,sum)
}

func verifyMult(number int) bool{
	if number % 3 == 0 || number % 5 == 0{
		return true
	}else{
		return false
	}
}